sample/src/main/java/com/smileidentity/sample/repo/DataStoreRepository.kt

```diff
@@ -70,6 +70,16 @@ object DataStoreRepository {
         mainDataStore.edit { it.clear() }
     }
 
+    /**
+     * Get the Smile string Config from [mainDataStore], if one has been manually set. (It may not be set if
+     * the sample app is utilizing the assets/smile_config.json file instead).
+     *
+     * If no config is set, this returns null
+     */
+    fun getStringConfig(): Flow<String?> = mainDataStore.data.map { preferences ->
+        preferences[Keys.config] ?: ""
```

Comment:
Why return an empty String here? That would not conform to Config json schema

```suggestion
        preferences[Keys.config]
```